build/cmake/DefineOptions.cmake

```diff
@@ -114,7 +104,11 @@ if(BUILD_CPP)
         list(APPEND boost_components unit_test_framework filesystem chrono program_options)
     endif()
     if(boost_components)
-        find_package(Boost 1.53 REQUIRED COMPONENTS ${boost_components})
+        hunter_add_package(Boost COMPONENTS ${boost_components})
```

Comment:
>  And the boost library are also not copied.

To clarify, Boost should be installed to the Hunter root folder. That is the command to trigger Boost build.